src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -43,14 +43,13 @@ object Boot extends App {
         swaggerConfig.getString("licenseUrl"))
       ))
 
-    //val storageDir = Paths.get(conf.getString("workspace.storageDir"))
-    //Files.createDirectories(storageDir)
-    //val workspaceDAO = new FileSystemWorkspaceDAO(storageDir)
-    //val entityDAO = NoOpEntityDAO
-    val backingDB = new OrientGraph("memory:rawls") // TODO: use orientdb-dev
```

Comment:
I assume the connection url, username, password, etc would be pulled from a config file here?